packages/quantic/force-app/main/default/lwc/quanticCaseClassification/__tests__/quanticCaseClassification.test.js

```diff
@@ -0,0 +1,848 @@
+            );
+            expect(allOptionsSelectInput.value).toBe(
+              exampleCaseClassificationSuggestions[0].value
+            );
+            expect(allOptionsSelectInput.placeholder).toBe(
+              exampleMoreTopicsLabel
+            );
+          });
+
+          describe('when selecting an option from the all options select input', () => {
+            it('should select correctly', async () => {
+              const exampleSelectedOptionIndex = 3;
+
+              const element = createTestComponent({
+                ...defaultOptions,
+                maxSuggestions: exampleMaxSuggestions,
+              });
+              mockGetPicklistValuesByRecordTypeWire();
+
+              await flushPromises();
+
+              const showSelectInputButton = element.shadowRoot.querySelector(
+                selectors.showSelectInputButton
+              );
+              showSelectInputButton.click();
+              await flushPromises();
+
+              const allOptionsSelectInput = element.shadowRoot.querySelector(
+                selectors.allOptionsSelectInput
+              );
+
+              expect(allOptionsSelectInput).not.toBeNull();
+              allOptionsSelectInput.value =
+                exampleCaseClassificationAllOptions[
+                  exampleSelectedOptionIndex
+                ].value;
+              allOptionsSelectInput.dispatchEvent(new CustomEvent('change'));
+
+              expect(functionsMocks.update).toHaveBeenCalledTimes(2);
+              expect(functionsMocks.update).toHaveBeenCalledWith(
+                exampleCaseClassificationAllOptions[exampleSelectedOptionIndex]
+                  .value,
+                {caseClassifications: false, documentSuggestions: false},
+                undefined
+              );
+            });
+          });
+        });
+      });
+    });
+
+    describe('when the max suggestions value is superior to the count of all the classification options', () => {
+      const exampleMaxSuggestions = 6;
+
+      describe('the case classification suggestions', () => {
+        it('should display the case classification suggestions as inline options', async () => {
+          const element = createTestComponent({
+            ...defaultOptions,
+            maxSuggestions: exampleMaxSuggestions,
+          });
+          mockGetPicklistValuesByRecordTypeWire();
+
+          await flushPromises();
+
+          const suggestions = element.shadowRoot.querySelectorAll(
+            selectors.caseClassificationSuggestion
+          );
+
+          expect(suggestions.length).toBe(
+            exampleCaseClassificationSuggestions.length
+          );
+          for (
+            let index = 0;
+            index < exampleCaseClassificationSuggestions.length;
+            index++
+          ) {
+            expect(suggestions[index].textContent).toBe(
+              exampleCaseClassificationSuggestions[index].value
+            );
+          }
```

Comment:
```suggestion
exampleCaseClassificationSuggestions.forEach((exampleSuggestion, index) => expect(suggestions[index].textContent).toBe(exampleSuggestion.value));
```

I think you might even be able to de-construct although I don't know if it's allowed in the forEach?

```suggestion
exampleCaseClassificationSuggestions.forEach(({value: expectedValue}, index) => expect(suggestions[index].textContent).toBe(expectedValue));
```

Reply:
Yup it works :D

Reply:
addressed!